plugins/README.md

```diff
@@ -72,6 +76,16 @@ exports.teardown = function(config) {};
  * @return Return values are ignored.
  */
 exports.postResults = function(config) {};
+
+/**
+ * Called after each test block completes.
+ *
+ * @param {Object} config The plugin configuration object.
+ *
+ * @return Object If an object is returned, it is merged with the Protractor
+ *     result object. May return a promise.
+ */
+exports.postTest = function(config) {};
```

Comment:
I think the problem is that these terms are overloaded and there's no general agreement on what they mean. "Test case" might mean one 'it' block or a whole file, or maybe even a whole set of things.

Do you prefer `spec`?